'LiveServer' object has no attribute 'url'

**Describe the bug**
When using `live_server` mode, the actual `live_server` fixture cannot be repr'd because of the missing `url` attribute.

This appears to have been broken in e9b8321b3e1051b163ad5342db0aa529e5f7ce9f, which removed the `url()` method

**To Reproduce**
Add `pytest-flask` to a project and attempt to print the `live_server` fixture, as so:

```
import pytest
import flask


@pytest.fixture(scope='session')
def app():
    return flask.Flask('test')

def test_foo(client, live_server):
    print(live_server)
```

**Expected behavior**
Pretty print

**Actual Behavior**
```
self = <[AttributeError("'LiveServer' object has no attribute 'url'") raised in repr()] LiveServer object at 0x7f0c7b63c6a0>

    def __repr__(self):
>       return "<LiveServer listening at %s>" % self.url()
E       AttributeError: 'LiveServer' object has no attribute 'url'
```

**Environment (please complete the following information):**
 - OS: Linux
 - Python Version: all
 - pytest-flask version: 1.2.0

--- a/pytest_flask/live_server.py
+++ b/pytest_flask/live_server.py
@@ -69,6 +69,12 @@ class LiveServer:
             sock.close()
         return ret
 
+    def url(self, url=""):
+        """Returns the complete url based on server options."""
+        return "http://{host!s}:{port!s}{url!s}".format(
+            host=self.host, port=self.port, url=url
+        )
+
     def stop(self):
         """Stop application process."""
         if self._process:

--- a/tests/test_live_server.py
+++ b/tests/test_live_server.py
@@ -12,6 +12,7 @@ class TestLiveServer:
     def test_init(self, live_server):
         assert live_server.port
         assert live_server.host == "localhost"
+        assert live_server.url() == "http://localhost:{0}".format(live_server.port)
 
     def test_server_is_alive(self, live_server):
         assert live_server._process

